### PR TITLE
util-macros: add util-macros package

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: recipes-debian/xorg-util/util-macros_1.19.2.bb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# base recipe: meta/recipes-graphics/xorg-util/util-macros_1.19.2.bb
```
Comment (masami256) [inline recipes-debian/xorg-util/util-macros_1.19.2.bb]: @yuwata The file name recipes-debian/xorg-util/util-macros_1.19.2.bb should use _debian suffix doesn't it?

**Comment by masami256** [comment]
@yuwata I got build error when try to build core-image-weston. xorgproto package build fails with this pr. 

This is build log.

```
Loading cache: 100% |##################################################################################################################################################################| Time: 0:00:00
Loaded 2007 entries from dependency cache.                                                                                                                                                            
WARNING: No bb files matched BBFILE_PATTERN_emlinux ''                                                                                                                                                
NOTE: Resolving any missing task queue dependencies                                                                                                                                                   
                                                                                                                                                                                                      
Build Configuration:                                                                                                                                                                                  
BB_VERSION           = "1.42.0"                                                                                                                                                                       
BUILD_SYS            = "x86_64-linux"                                                                                                                                                                 
NATIVELSBSTRING      = "ubuntu-18.04"                                                                                                                                                                 
TARGET_SYS           = "aarch64-emlinux-linux"                                                                                                                                                        
MACHINE              = "qemuarm64"                                                                                                                                                                    
DISTRO               = "emlinux"                                                                                                                                                                      
DISTRO_VERSION       = "2.0"                                                                                                                                                                          
TUNE_FEATURES        = "aarch64 armv8a crc"                                                                                                                                                           
TARGET_FPU           = ""                                                                                                                                                                             
meta                 = "HEAD:6d2e12e79211b31cdf5ea824fb9a8be54ba9a9eb"                                                                                                                                
meta-debian          = "HEAD:51616e210b11bed313a9a805b1f50d8cba0746a8"                                                                                                                                
meta-debian-extended = "test-162:bb62126ae06576a4b82796b6056722f45fd6fac7"                                                                                                                            
meta-emlinux         = "HEAD:a78096888cf2afeb72ebe2413d6df8e55074214a"                                                                                                                                
                                                                                                                                                                                                      
NOTE: Tainting hash to force rebuild of task /home/masami/pr-test/repos/meta-debian-extended/recipes-debian/xorg-proto/xorgproto_debian.bb, do_build                                   | ETA:  0:00:00
WARNING: /home/masami/pr-test/repos/meta-debian-extended/recipes-debian/xorg-proto/xorgproto_debian.bb.do_build is tainted from a forced run                                           | ETA:  0:00:00
Initialising tasks: 100% |#############################################################################################################################################################| Time: 0:00:00
Sstate summary: Wanted 10 Found 4 Missed 6 Current 76 (40% match, 93% complete)                                                                                                                       
NOTE: Executing SetScene Tasks                                                                                                                                                                        
NOTE: Executing RunQueue Tasks                                                                                                                                                                        
ERROR: xorgproto-2018.4-r0 do_configure: autoreconf execution failed.                                                                                                                                 
ERROR: xorgproto-2018.4-r0 do_configure: Function failed: do_configure (log file is located at /home/masami/pr-test/build/tmp-glibc/work/aarch64-emlinux-linux/xorgproto/2018.4-r0/temp/log.do_configu
re.15413)                                                                                                                                                                                             
ERROR: Logfile of failure stored in: /home/masami/pr-test/build/tmp-glibc/work/aarch64-emlinux-linux/xorgproto/2018.4-r0/temp/log.do_configure.15413
Log data follows:
| DEBUG: SITE files ['endian-little', 'bit-64', 'arm-common', 'arm-64', 'common-linux', 'common-glibc', 'aarch64-linux', 'common']
| DEBUG: Executing shell function autotools_preconfigure
| DEBUG: Shell function autotools_preconfigure finished
| DEBUG: Executing python function autotools_aclocals
| DEBUG: SITE files ['endian-little', 'bit-64', 'arm-common', 'arm-64', 'common-linux', 'common-glibc', 'aarch64-linux', 'common']
| DEBUG: Python function autotools_aclocals finished
| DEBUG: Executing shell function do_configure
| automake (GNU automake) 1.16.1
| Copyright (C) 2018 Free Software Foundation, Inc.
| License GPLv2+: GNU GPL version 2 or later <https://gnu.org/licenses/gpl-2.0.html>
| This is free software: you are free to change and redistribute it.
| There is NO WARRANTY, to the extent permitted by law.
| 
| Written by Tom Tromey <tromey@redhat.com>
|        and Alexandre Duret-Lutz <adl@gnu.org>.
| AUTOV is 1.16
| NOTE: Executing ACLOCAL="aclocal --system-acdir=/home/masami/pr-test/build/tmp-glibc/work/aarch64-emlinux-linux/xorgproto/2018.4-r0/recipe-sysroot/usr/share/aclocal/ --automake-acdir=/home/masami/
pr-test/build/tmp-glibc/work/aarch64-emlinux-linux/xorgproto/2018.4-r0/recipe-sysroot-native/usr/share/aclocal-1.16" autoreconf -Wcross --verbose --install --force --exclude=autopoint -I /home/masam
i/pr-test/build/tmp-glibc/work/aarch64-emlinux-linux/xorgproto/2018.4-r0/recipe-sysroot-native/usr/share/aclocal/
| autoreconf: Entering directory `.'
| autoreconf: configure.ac: not using Gettext
| autoreconf: running: aclocal --system-acdir=/home/masami/pr-test/build/tmp-glibc/work/aarch64-emlinux-linux/xorgproto/2018.4-r0/recipe-sysroot/usr/share/aclocal/ --automake-acdir=/home/masami/pr-t
est/build/tmp-glibc/work/aarch64-emlinux-linux/xorgproto/2018.4-r0/recipe-sysroot-native/usr/share/aclocal-1.16 -I /home/masami/pr-test/build/tmp-glibc/work/aarch64-emlinux-linux/xorgproto/2018.4-r0
/recipe-sysroot-native/usr/share/aclocal/ --force
| configure.ac:30: error: must install xorg-macros 1.12 or later before running autoconf/autogen
| configure.ac:30: the top level
| autom4te: m4 failed with exit status: 1
| aclocal: error: echo failed with exit status: 1
| autoreconf: aclocal failed with exit status: 1
| ERROR: autoreconf execution failed.
| WARNING: exit code 1 from a shell command.
| ERROR: Function failed: do_configure (log file is located at /home/masami/pr-test/build/tmp-glibc/work/aarch64-emlinux-linux/xorgproto/2018.4-r0/temp/log.do_configure.15413)
ERROR: Task (/home/masami/pr-test/repos/meta-debian-extended/recipes-debian/xorg-proto/xorgproto_debian.bb:do_configure) failed with exit code '1'
NOTE: Tasks Summary: Attempted 593 tasks of which 585 didn't need to be rerun and 1 failed.

Summary: 1 task failed:
  /home/masami/pr-test/repos/meta-debian-extended/recipes-debian/xorg-proto/xorgproto_debian.bb:do_configure
Summary: There were 2 WARNING messages shown.
Summary: There were 2 ERROR messages shown, returning a non-zero exit code.
```

This is log.do_configure.

```
EBUG: SITE files ['endian-little', 'bit-64', 'arm-common', 'arm-64', 'common-linux', 'common-glibc', 'aarch64-linux', 'common']
DEBUG: Executing shell function autotools_preconfigure
DEBUG: Shell function autotools_preconfigure finished
DEBUG: Executing python function autotools_aclocals
DEBUG: SITE files ['endian-little', 'bit-64', 'arm-common', 'arm-64', 'common-linux', 'common-glibc', 'aarch64-linux', 'common']
DEBUG: Python function autotools_aclocals finished
DEBUG: Executing shell function do_configure
automake (GNU automake) 1.16.1
Copyright (C) 2018 Free Software Foundation, Inc.
License GPLv2+: GNU GPL version 2 or later <https://gnu.org/licenses/gpl-2.0.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by Tom Tromey <tromey@redhat.com>
       and Alexandre Duret-Lutz <adl@gnu.org>.
AUTOV is 1.16
NOTE: Executing ACLOCAL="aclocal --system-acdir=/home/masami/pr-test/build/tmp-glibc/work/aarch64-emlinux-linux/xorgproto/2018.4-r0/recipe-sysroot/usr/share/aclocal/ --automake-acdir=/home/masami/pr-test/build/tmp-glibc/work/aarch64-emlinux-linux/xorgproto/2018.4-r0/recipe-sysroot-native/usr/share/aclocal-1.16" autoreconf -Wcross --verbose --install --force --exclude=autopoint -I /home/masami/pr-test/build/tmp-glibc/work/aarch64-emlinux-linux/xorgproto/2018.4-r0/recipe-sysroot-native/usr/share/aclocal/
autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --system-acdir=/home/masami/pr-test/build/tmp-glibc/work/aarch64-emlinux-linux/xorgproto/2018.4-r0/recipe-sysroot/usr/share/aclocal/ --automake-acdir=/home/masami/pr-test/build/tmp-glibc/work/aarch64-emlinux-linux/xorgproto/2018.4-r0/recipe-sysroot-native/usr/share/aclocal-1.16 -I /home/masami/pr-test/build/tmp-glibc/work/aarch64-emlinux-linux/xorgproto/2018.4-r0/recipe-sysroot-native/usr/share/aclocal/ --force
configure.ac:30: error: must install xorg-macros 1.12 or later before running autoconf/autogen
configure.ac:30: the top level
autom4te: m4 failed with exit status: 1
aclocal: error: echo failed with exit status: 1
autoreconf: aclocal failed with exit status: 1
ERROR: autoreconf execution failed.
WARNING: exit code 1 from a shell command.
ERROR: Function failed: do_configure (log file is located at /home/masami/pr-test/build/tmp-glibc/work/aarch64-emlinux-linux/xorgproto/2018.4-r0/temp/log.do_configure.15413)                                                                                                                                                                              
```
Build success without this pr.
 
```
Loading cache: 100% |##################################################################################################################################################################| Time: 0:00:00
Loaded 2007 entries from dependency cache.                                                                                                                                                            
WARNING: No bb files matched BBFILE_PATTERN_emlinux ''                                                                                                                                                
NOTE: Resolving any missing task queue dependencies                                                                                                                                                   
                                                                                                                                                                                                      
Build Configuration:                                                                                                                                                                                  
BB_VERSION           = "1.42.0"                                                                                                                                                                       
BUILD_SYS            = "x86_64-linux"                                                                                                                                                                 
NATIVELSBSTRING      = "ubuntu-18.04"                                                                                                                                                                 
TARGET_SYS           = "aarch64-emlinux-linux"                                                                                                                                                        
MACHINE              = "qemuarm64"                                                                                                                                                                    
DISTRO               = "emlinux"                                                                                                                                                                      
DISTRO_VERSION       = "2.0"                                                                                                                                                                          
TUNE_FEATURES        = "aarch64 armv8a crc"                                                                                                                                                           
TARGET_FPU           = ""                                                                                                                                                                             
meta                 = "HEAD:6d2e12e79211b31cdf5ea824fb9a8be54ba9a9eb"                                                                                                                                
meta-debian          = "HEAD:51616e210b11bed313a9a805b1f50d8cba0746a8"                                                                                                                                
meta-debian-extended = "warrior:e23aae320072d0ad390fb56fa54a0a42a8b1b2ff"                                                                                                                             
meta-emlinux         = "HEAD:a78096888cf2afeb72ebe2413d6df8e55074214a"                                                                                                                                
                                                                                                                                                                                                      
NOTE: Tainting hash to force rebuild of task /home/masami/pr-test/repos/meta-debian-extended/recipes-debian/xorg-proto/xorgproto_debian.bb, do_build                                   | ETA:  0:00:00
WARNING: /home/masami/pr-test/repos/meta-debian-extended/recipes-debian/xorg-proto/xorgproto_debian.bb.do_build is tainted from a forced run                                           | ETA:  0:00:00
Initialising tasks: 100% |#############################################################################################################################################################| Time: 0:00:00
Sstate summary: Wanted 10 Found 4 Missed 6 Current 76 (40% match, 93% complete)                                                                                                                       
NOTE: Executing SetScene Tasks                                                                                                                                                                        
NOTE: Executing RunQueue Tasks                                                                                                                                                                        
NOTE: Tasks Summary: Attempted 599 tasks of which 583 didn't need to be rerun and all succeeded.                                                                                                      
                                                                                                                                                                                                      
Summary: There were 2 WARNING messages shown.                                          
```

File: recipes-debian/xorg-util/util-macros_1.19.2.bb (hunk 1 above)
Comment (yuwata) [reply recipes-debian/xorg-util/util-macros_1.19.2.bb]: Ouch! Fixed.

**Comment by yuwata** [comment]
> @yuwata I got build error when try to build core-image-weston. xorgproto package build fails with this pr.

Indeed... Will take a look.

**Comment by yuwata** [comment]
@masami256 Updated. PTAL.

**Comment by masami256** [comment]
LGTM!

**Pull request merged**